frontend/src/stores/user.js

```diff
@@ -76,10 +76,24 @@ class userStore {
         this.balance = userData.balance;
         this.phoneNumber = userData.phoneNumber;
     }
+    resetCredentials() {
+        this.token = null;
+        this.username = null;
+        this.balance = null;
+        this.phoneNumber = null;
+    }
     setBalance(amount) {
         if (amount > this.balance) throw new Error('Insufficient fund!');
         else this.balance = this.balance - amount;
     }
+    async logout() {
+        try {
+            logoutApi();
```

Comment:
I think it would result in cleaner solution if these were imported like 

```
import * as backendApi from './backend';
```

and then used 

```
backendApi.logout();
```

Reply:
it's just choice of styles and does not have any significant impact on the applications.